app/domain/logs.rb

```diff
@@ -14,6 +14,26 @@ module Conjur
       code: "CONJ00038I"
     )
 
+    FetchingTrustedProxies = ::Util::TrackableLogMessageClass.new(
+        msg:  "Fetching trusted proxies list from {0-host-id}",
+        code: "CONJ00039D"
+    )
+
+    FetchedTrustedProxies = ::Util::TrackableLogMessageClass.new(
+        msg:  "{0-list-length} trusted proxies ip addresses has fetched",
```

Comment:
Let's discuss...

Reply:
@shulifink this is the amount of IP addresses that fetched, what do you want to discuss about?